monetio/models/_camx_mm.py

```diff
@@ -448,15 +448,18 @@ def _calc_midlayer_height_agl(dset):
     else:
         raise "No height variable found, but _calc_midlayer_height_agl was called."
     mid_layer_height = np.array(dset[height])  # height in the layer upper interface of each layer
-    layer_height = dset[height]
+    layer_height_agl = dset[height]
+    layer_height_agl.attrs["long_name"] = "layer_height_agl"
```

Comment:
`long_name` shouldn't generally be the same as `name` (if so no need to add it)

Reply:
Hopefully is better now.